assignment2/view/UserApplication.java

```diff
@@ -0,0 +1,61 @@
+package assignment2.view;
+
+import java.util.Scanner;
+import java.util.Set;
+import java.util.TreeSet;
+
+import assignment2.UserManager;
+import assignment2.data.User;
+import assignment2.exceptions.InvalidUserDetailException;
+import assignment2.util.DiskManager;
+
+public class UserApplication{
+	public static void init() {
+		Set<User> users = new TreeSet<User>();
+		//get the user from the disk if saved previously
+		users= DiskManager.getUsers();
```

Comment:
this could be directly initialised, rather than using 2 lines for initialisation.